LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Airbnb
+Copyright (c) 2017-present Airbnb
```

Comment:
The copyright didn't start in 2017; this should just say "2012". The " - present" is always implied and thus is redundant.